customer-terminate-service/app/src/main/java/se/inera/intyg/cts/application/service/TerminationService.java

```diff
@@ -50,6 +58,32 @@ public TerminationDTO create(CreateTerminationDTO createTerminationDTO) {
     return toDTO(createdTermination);
   }
 
+  public TerminationDTO update(UUID terminationId, TerminationDTO terminationDTO) {
+    if (!terminationId.equals(terminationDTO.terminationId())) {
```

Comment:
Feel a bit redundant to pass the same parameter 2 times to this method. Unless we can update all the fields in TerminationDTO we should not use it as a parameter. (Now we only update/use 4 of 10 fields/arguments provided.)

Reply:
I didn't want to put that logic in the controller, instead this app service verifies the data received through the api and then calls the domain service with the fields to update. If in tomorrow we allow another field to be updated, we don't have to change the api.

Reply:
With this solution we allow the caller to provide arguments that are not used but can be provided. This make the API unsafe as we can not trust the service to updated all fields we provide. Another solution would be to use a updateTerminationDTO what only contains the 3 fields that will be used and give us the option to add optional fields in the future that will not affect the API.